packages/react-components/react-tag-picker/stories/src/TagPicker/TagPickerNoPopover.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import {
+  TagPicker,
+  TagPickerInput,
+  TagPickerControl,
+  TagPickerProps,
+  TagPickerGroup,
+} from '@fluentui/react-components';
+import { Tag, Avatar, Field } from '@fluentui/react-components';
+
+export const NoPopover = () => {
+  const [selectedOptions, setSelectedOptions] = React.useState<string[]>([]);
+
+  const onOptionSelect: TagPickerProps['onOptionSelect'] = (_, data) => {
+    setSelectedOptions(data.selectedOptions);
+  };
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setInputValue(event.currentTarget.value);
+  };
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === 'Enter' && inputValue) {
+      setInputValue('');
+      setSelectedOptions(curr => [...curr, inputValue]);
+    }
+  };
+
+  const [inputValue, setInputValue] = React.useState('');
+
+  return (
+    <Field label="Add Employees" style={{ maxWidth: 400 }}>
+      <TagPicker noPopover onOptionSelect={onOptionSelect} selectedOptions={selectedOptions}>
+        <TagPickerControl expandIcon={null}>
```

Comment:
can the tagpicker control infer from context that there is no popover and then just not render the expand icon?